format/Format.py

```diff
@@ -112,6 +112,10 @@ def understand(image_file):
     def ignore(cls):
         return False
 
+    @classmethod
+    def is_abstract(cls):
```

Comment:
Possibly worth adding a docstring which explains what is meant by abstract in this case e.g. will be ignored by the registry if identified as the only supporting class

Reply:
Also `is_abstract()` -> `abstract_format()` maybe?